tsdb/engine/tsm1/engine_test.go

```diff
@@ -70,50 +70,6 @@ func Test_DevEngineCacheQueryAscending(t *testing.T) {
 	}
 }
 
-// Ensure an engine containing cached values responds correctly to queries.
-func Test_DevEngineCacheQueryDescending(t *testing.T) {
```

Comment:
ah, for some reason I thought this was testing that the `CacheKeyIterator` would return things in descending order. I removed it because that's no longer its responsibility, but I see now that it's actually grabbing a cursor object there. Will put back in.